config/thanos/manifests/thanos-query-serviceMonitor.yaml

```diff
@@ -0,0 +1 @@
+null
```

Comment:
needed?

Reply:
No, but these files in the manifests directory are generated.
I can remove them, but if we regen they'll have to be removed again.
They don't seem to cause issue with kustomize

I'm happy to remove them now if you want

Reply:
I updated the jsonnet file to generate these 2 files since I don't see any reason not to.
Changes pushed.